src/main/resources/openapi/GarminAPI.yaml

```diff
@@ -94,12 +151,26 @@ components:
           items:
             type: string
           description: List of permission types
+          default: [ ]
         changeTimeInSeconds:
           type: integer
           format: int64
           description: Timestamp of when the permissions changed
+          default: 0
       required:
         - userId
         - summaryId
-        - permissions
-        - changeTimeInSeconds
\ No newline at end of file
+
+
+    SummariesRequest:
+      type: object
+      additionalProperties:
+        type: array
+        items:
+          allOf:
+            - type: object
+              additionalProperties: true
+      description: |
+        Dynamic object where each property is an array of summary data.
+        Property names can be: dailies, epochs, hrv, sleeps, pulseox, stressDetails, or any other summary type.
+        Each item in the array must contain userId and can have any additional properties.
```

Comment:
Willst Du das in Zukunft noch besser ausbauen - also dailies, epochs, hrv, sleeps, pulseox, stressDetails modellieren - oder ist der Plan das als RAW zu übernehmen?